libs/auth/src/java_card.ts

```diff
@@ -854,24 +858,61 @@ export class JavaCard implements Card {
    * Creates and stores the card's VotingWorks-issued cert. Used by the initial card configuration
    * script.
    */
-  async createAndStoreCardVxCert(): Promise<void> {
+  async createAndStoreCardVxCert(remoteCertParams?: {
+    randomId: string;
+    workingDirectory: string;
+  }): Promise<void> {
     assert(this.cardProgrammingConfig !== undefined);
     assert(this.cardProgrammingConfig.configType === 'vx');
     await this.selectApplet();
 
-    const publicKey = await this.generateAsymmetricKeyPair(
-      CARD_VX_CERT.PRIVATE_KEY_ID
-    );
-    const cardVxCert = await createCert({
-      certKeyInput: {
-        type: 'public',
-        key: { source: 'inline', content: publicKey.toString('utf-8') },
-      },
-      certSubject: constructCardCertSubjectWithoutJurisdictionAndCardType(),
-      expiryInDays: CERT_EXPIRY_IN_DAYS.CARD_VX_CERT,
-      signingCertAuthorityCertPath: this.vxCertAuthorityCertPath,
-      signingPrivateKey: this.cardProgrammingConfig.vxPrivateKey,
-    });
-    await this.storeCert(CARD_VX_CERT.OBJECT_ID, cardVxCert);
+    const certPublicKey = (
+      await this.generateAsymmetricKeyPair(CARD_VX_CERT.PRIVATE_KEY_ID)
+    ).toString('utf-8');
+
+    let cert: Buffer;
+    if (this.cardProgrammingConfig.vxPrivateKey.source === 'remote') {
+      assert(remoteCertParams !== undefined);
+      const { randomId, workingDirectory } = remoteCertParams;
+      const certPublicKeyPath = path.join(
+        workingDirectory,
+        `public-key-${randomId}.pem`
+      );
+      const certPath = path.join(workingDirectory, `cert-${randomId}.pem`);
+
+      await fs.writeFile(certPublicKeyPath, certPublicKey);
+
+      const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+      });
+      await rl.question(`A card-specific public key has been written to ${certPublicKeyPath}.
+Share this file with someone who has the appropriate signing key.
+They will return to you a ${path.basename(certPath)} file.
+Place that file in ${workingDirectory} and press enter. `);
```

Comment:
You'll see in the next commit the script that'll turn this card public key into a card cert